worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -12,24 +12,25 @@ prompt where you can read information about your run and write any command to in
 
 The randomization is performed by the AP.json file, an output file generated by the Archipelago server.
 
-## Installation Procedures
+## Installation Procedures  
 
-**This client has only been tested with the Official Steam version of the game (v1.15/1.35) not matter which DLCs are installed**
+<span style="color:red">**This mod can ban you permanently from the FromSoftware servers if used online** </span>.  
```

Comment:
```suggestion
<span style="color:Tomato">
**This mod can ban you permanently from the FromSoftware servers if used online.** 
</span>  
```
red (`#ff0000 `) is super hard to read on the grass background. Tomato looks still red but has okay-ish contrast.
When viewing in source form, it's better to have the tags on separate lines if possible.